cli/sawtooth_cli/cluster.py

```diff
@@ -252,7 +252,12 @@ def do_cluster_start(args):
     state["DesiredState"] = "Running"
 
     if args.processors is None:
-        raise CliException("Use -P to specify one or more processors")
+        error_string = "Use -P to specify one or more processors.\n"
+        error_string += "Included processors are found in sawtooth-core/bin " \
```

Comment:
Is that where they will be if the system is installed rather than run from a dev environment?